tools/protocol-schema-check/src/main.rs

```diff
@@ -0,0 +1,121 @@
+/// Needed for `inventory`, helps to register all `ProtocolStruct`-ed structs.
+/// This is needed to be able to iterate over all structs and check their
+/// hashes.
+/// Iteration is done by `ProtocolStructInfo`s generated by `ProtocolStruct`
+/// macro.
+// TODO (#11755): add to CI.
+
+// Needed because otherwise tool doesn't notice `ProtocolStructInfo`s from
+// other crates.
+#[allow(unused_imports)]
+use near_primitives::*;
+
+use near_structs_checker_core::ProtocolStructInfo;
+use std::any::TypeId;
+use std::collections::hash_map::DefaultHasher;
+use std::collections::{BTreeMap, HashSet};
+use std::fs;
+use std::hash::{Hash, Hasher};
+use std::path::Path;
+
+#[allow(unused)]
+fn compute_hash(
+    info: &ProtocolStructInfo,
+    structs: &BTreeMap<TypeId, &'static ProtocolStructInfo>,
+) -> u32 {
+    let mut hasher = DefaultHasher::new();
```

Comment:
Still using `DefaultHasher` here.

Reply:
Yeah I didn't realise I need to fix that as well.